rest/route/github.go

```diff
@@ -583,6 +589,102 @@ func (gh *githubHookApi) AddIntentForPR(pr *github.PullRequest, owner, calledBy
 		}
 	}
 
+	patches, err := patch.Find(patch.ByGithash(pr.Head.GetSHA()))
+	if err != nil {
+		grip.Info(message.Fields{
+			"message":           "error getting same hash patches",
+			"owner":             pr.Base.User.GetLogin(),
+			"repo":              pr.Base.Repo.GetName(),
+			"ref":               pr.Head.GetRef(),
+			"pr_num":            pr.GetNumber(),
+			"hash":              pr.Head.GetSHA(),
+			"override_existing": overrideExisting,
+			"err":               err.Error(),
+		})
+		return errors.Wrapf(err, "getting same hash patches")
+	}
+	// Remove a patch associated with this PR if applicable.
+	var filtered = []patch.Patch{}
```

Comment:
nit: this is equivalent to `filtered := []patch.Patch{}`, so the `var` can be omitted. Alternatively, it seems like your logic doesn't really care if the slice is nil vs empty (both would be treated the same in this function), so you could just do `var filtered []patch.Patch` to have a nil unallocated slice, which is auto-allocated when it's `append`ed to.